examples/README.md

```diff
@@ -50,6 +50,41 @@ kubectl replace -f ./csi-config-map-sample.yaml
 Storage class and snapshot class, using `<cluster-id>` as the value for the
 option `clusterID`, can now be created on the cluster.
 
+## Running CephCSI with pod networking
+
+The current problem with Pod Networking, is when a CephFS/RBD volume is mounted
+in a pod using Ceph CSI and then the CSI CephFS/RBD plugin is restarted or
+terminated (e.g. by restarting or deleting its DaemonSet), all operations on
+the volume become blocked, even after restarting the CSI pods.
+
+The only workaround is to restart the node where the Ceph CSI plugin pod was
+restarted.  This can be mitigated by running the `rbd map`/`mount -t` commands
+in a different namespace which does not get deleted and recreated often.
+
+If someone wants to run the CephCSI with the pod networking they can still do
+by setting the `netNamespaceFilePath`. If this path is set CephCSI will execute
+the `rbd map`/`mount -t` commands by entering the [network
+namespace](https://man7.org/linux/man-pages/man7/network_namespaces.7.html)
+specified by `netNamespaceFilePath` with the
+[nsenter](https://man7.org/linux/man-pages/man1/nsenter.1.html) command.
+
+`netNamespaceFilePath` is typical of the network namespace of a long-running
+process it usually `/proc/<long running process id>/ns/net`
+
+The long-running process can also be another pod which is a (Daemonset) which
+never restarts. This Pod should only be stopped and restarted when a node is
+stopped so that volume operations do not become blocked. The new DaemonSet pod
+can contain a single container, the container responsible for holding
+filesystem mounts and mapped block devices. It is used as a passthrough by the
```

Comment:
It wouldn't "hold" mounts or block devices -- in fact they don't need holding.  It's all about the pod network:
```suggestion
can contain a single container, responsible for holding its pod network alive.
It is used as a passthrough by the
```